assets/js/modules/adsense/util.js

```diff
@@ -181,7 +181,8 @@ export async function getAdSenseAccountStatus( statusUpdateCallback, existingTag
 				for ( let account of accounts ) {
 					const accountId = account.id;
 					const urlchannels = await data.get( 'modules', 'adsense', 'urlchannels', { clientId: accountId } ).then( res => res ).catch( e => e );
-					const matches = urlchannels && urlchannels.length ? filter( urlchannels, { urlPattern: googlesitekit.admin.siteURL } ) : [];
+					const parsedUrl = new URL( googlesitekit.admin.siteURL );
+					const matches = urlchannels && urlchannels.length ? filter( urlchannels, { urlPattern: parsedUrl.hostname } ) : [];
```

Comment:
Can you provide more background on this change?

Reply:
AdSense url pattern only includes the hostname, no http/https, it would've never matched googlesitekit.admin.siteURL in this case, you would've come across this bug with more than 1 account.